### PR TITLE
Add error alert and logging

### PR DESCRIPTION
**Pull request by indhu-gunda**
image

### CODE REVIEW

File: backend/src/main/java/com/google/capstone/TextToSpeech.java [hunk 1]
```diff
@@ -67,14 +70,20 @@ private ByteString generateAudio(TextToSpeechClient textToSpeechClient, String t
...
     Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
     BlobId blobId = BlobId.of(BUCKET_NAME, objectIdString);
     BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("audio/mpeg").build();
-    storage.create(blobInfo, audioContents.toByteArray());
+    try {
+      storage.create(blobInfo, audioContents.toByteArray());
+    } catch (StorageException e) {
+      logger.severe("Audio file unable to be uploaded to Google Cloud Storage.");
+    }
+    logger.info("Audio file successfully uploaded to Google Cloud Storage.");
```
Comment (isabelsang) [inline backend/src/main/java/com/google/capstone/TextToSpeech.java:86]: Where does this logger.info go? From line 34 I see it's related to Text-to-Speech so is it read out?

Comment (indhu-gunda) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:86]: This is just information that will be in the logs. Line 34 initializes the logger for this class.

File: backend/src/main/java/com/google/capstone/TextToSpeech.java [hunk 2]
```diff
@@ -11,10 +12,7 @@
 import com.google.appengine.api.blobstore.BlobKey;
 import com.google.appengine.api.blobstore.BlobstoreService;
 import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
-import com.google.cloud.storage.BlobId;
-import com.google.cloud.storage.BlobInfo;
-import com.google.cloud.storage.Storage;
-import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.*;
```
Comment (katch00) [inline backend/src/main/java/com/google/capstone/TextToSpeech.java:15]: Style nit: Style guide says not to use wildcard imports :(

File: backend/src/main/java/com/google/capstone/TextToSpeech.java [hunk 3]
```diff
@@ -49,10 +52,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.setContentType("text/plain");
         response.getWriter().println(blobKey);
       } catch (IOException e) {
-        response.sendError(500, "Unable to generate audio file.");
+        String errorMsg = "Unable to generate audio file.";
+        logger.severe(errorMsg);
```
Comment (BearBurg) [inline backend/src/main/java/com/google/capstone/TextToSpeech.java:56]: It seems that we can add more contexts in the logging message. Think about what information would help you when you are debugging. For example you can add the textString and objectIdString.

File: backend/src/main/java/com/google/capstone/TextToSpeech.java [hunk 4]
```diff
@@ -49,10 +52,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+        response.sendError(500, errorMsg);
       }
     } else {
-      response.sendError(400, "Invalid text or object id.");
+      String errorMsg = "Invalid text or object id.";
+      logger.severe(errorMsg);
```
Comment (BearBurg) [inline backend/src/main/java/com/google/capstone/TextToSpeech.java:61]: Same here.

File: backend/src/main/java/com/google/capstone/TextToSpeech.java (hunk 3 above)
Comment (indhu-gunda) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:58]: changed

File: backend/src/main/java/com/google/capstone/TextToSpeech.java (hunk 4 above)
Comment (indhu-gunda) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:64]: added

**Pull request merged**